Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -125,9 +127,13 @@
                               value="&lt;#{fn:length(fn:split(defaultTreeNode.rowKey, '_'))}&gt; "/>
                 <h:outputText value="#{empty logicalElementType ? msgs['dataEditor.withoutType'] : logicalElementType}"
                               title="#{logicalNode.undefined ? msgs['dataEditor.undefinedStructure'] : ''}"
-                              rendered="#{DataEditorForm.structurePanel.titleMetadata eq 'type'}"/>
+                              rendered="#{DataEditorForm.structurePanel.nodeLabelOption.contains('type') or logicalNode.linked}"/>
+                <h:outputText value=" - " 
+                              rendered="#{DataEditorForm.structurePanel.nodeLabelOption.contains('+') and !logicalNode.linked}" />
                 <h:outputText value="#{DataEditorForm.getStructureElementTitle(logicalNode.dataObject)}"
-                              rendered="#{DataEditorForm.structurePanel.titleMetadata ne 'type'}"/>
+                              rendered="#{DataEditorForm.structurePanel.nodeLabelOption.contains('title') and !logicalNode.linked}"/>
+                <!--<h:outputText value="1234"
```

Comment:
Does this comment have any particular purpose?

Reply:
No, sorry. I removed it.